bunq/sdk/client.py

```diff
@@ -117,6 +130,20 @@ def _request(self, method, uri_relative, request_bytes, custom_headers):
 
         return self._create_bunq_response_raw(response)
 
+    @classmethod
+    def _append_params_to_uri(cls, uri, params):
+        """
+        :type uri: str
+        :type params: dict[str, str]
+
+        :rtype: str
+        """
+
+        if params:
```

Comment:
Wouldn't this be always true?

Reply:
@OGKevin nope. Empty dictionaries evaluate to False! https://docs.python.org/2/library/stdtypes.html#truth-value-testing

Reply:
Ah right 👍